docs/root/configuration/overview/v2_overview.rst

```diff
@@ -332,6 +332,40 @@ The management server could respond to EDS requests with:
               address: 127.0.0.2
               port_value: 1234
 
+.. _config_overview_v2_extension_configuration:
+
+Extension configuration
+-----------------------
```

Comment:
Can you add some discussion in here of the UDPA typed struct also? In general I would like to see a discussion of all the various acceptable ways of configuring extensions and why a user might want to choose them.

Reply:
Sure. I didn't find any mention of typed struct, but can start sketching something in this section.